packages/xrpl/test/integration/utils.ts

```diff
@@ -1,45 +1,196 @@
 import { assert } from 'chai'
-import _ from 'lodash'
+import omit from 'lodash/omit'
+import throttle from 'lodash/throttle'
 import { decode } from 'ripple-binary-codec'
-import { Client, Wallet, AccountInfoRequest } from 'xrpl-local'
+import {
+  Client,
+  Wallet,
+  AccountInfoRequest,
+  type SubmitResponse,
+  TimeoutError,
+  NotConnectedError,
+  unixTimeToRippleTime,
+} from 'xrpl-local'
 import { Payment, Transaction } from 'xrpl-local/models/transactions'
 import { hashSignedTx } from 'xrpl-local/utils/hashes'
 
 const masterAccount = 'rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh'
 const masterSecret = 'snoPBrXtMeMyMHUVTgbuqAfg1SUTb'
 
-export async function ledgerAccept(client: Client): Promise<void> {
-  const request = { command: 'ledger_accept' }
-  await client.connection.request(request)
+async function sendLedgerAccept(client: Client): Promise<unknown> {
+  return client.connection.request({ command: 'ledger_accept' })
 }
 
-export function subscribeDone(client: Client, done: Mocha.Done): void {
+/**
+ * Throttles an async function in a way that can be awaited.
+ * By default throttle doesn't return a promise for async functions unless it's invoking them immediately.
+ *
+ * @param func - async function to throttle calls for.
+ * @param wait - same function as lodash.throttle's wait parameter. Call this function at most this often.
+ * @returns a promise which will be resolved/ rejected only if the function is executed, with the result of the underlying call.
+ */
+function asyncThrottle<F extends (...args: unknown[]) => Promise<unknown>>(
```

Comment:
I thought you said this didn't work? Does it work now? If it's not really doing anything, there's no reason to add additional complexity to the code.

Reply:
I swapped the throttle function and it seems to be working